### PR TITLE
[HOSS-170][MI] change accessibility to accessibility statement

### PR DESCRIPTION
**Pull request by muhammadibraaheem**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -103,3 +103,15 @@ shuttering.title=Service unavailable
...
 # Accessibility
 accessibility.title=Accessibility statement for check settled status
+
+# Footer Links
+footer.links.support_links.text=Support links
+footer.links.help_page.text=Help using GOV.UK
+footer.links.help_page.url=https://www.gov.uk/help
+footer.links.cookies.text=Cookies
+footer.links.cookies.url=/help/cookies
+footer.links.accessibility.text=Accessibility
```
Comment (russellthorn) [inline conf/messages:113]: Doesn't this need to be Accessibility statement?

File: app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
 @import uk.gov.hmrc.play.views.html.layouts._
 
 @this(
-  layoutsHead: Head, layoutsHeaderNav: HeaderNav, layoutsFooter: Footer, layoutsFooterLinks: FooterLinks,
+  layoutsHead: Head, layoutsHeaderNav: HeaderNav, layoutsFooter: Footer, layoutsFooterLinks: HossFooterLinks,
```
Comment (russellthorn) [inline app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html:25]: Does this work? Do you not need to import/reference (in the head of the file) the local helper?

Comment (udinobi) [reply app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html:25]: The `import` is not necessary as the templates, even after the compilation, are in the same folder

File: conf/messages [hunk 3]
```diff
@@ -110,7 +110,7 @@ footer.links.help_page.text=Help using GOV.UK
 footer.links.help_page.url=https://www.gov.uk/help
 footer.links.cookies.text=Cookies
 footer.links.cookies.url=/help/cookies
-footer.links.accessibility.text=Accessibility
+footer.links.accessibility.text=Accessibility statement
```
Comment (russellthorn) [inline conf/messages:113]: 👍

**Pull request merged**